lib/rules/quotes.js

```diff
@@ -14,6 +14,18 @@ export default class Quotes extends Rule {
       }
       case 'string': {
         if (['double', 'single'].includes(config)) {
+          return {
+            hbs: config,
+            html: config,
+          };
+        }
+        break;
+      }
+      case 'object': {
+        if (
+          ['double', 'single'].includes(config.hbs) &&
+          ['double', 'single'].includes(config.html)
+        ) {
           return config;
         }
```

Comment:
Can we throw an error if there are any unknown properties in the object? And add an error test case for this of course.

Reply:
Looks like this is still needed.